<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-1</y>
     <width>411</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    background: qradialgradient(
        cx:0.5, cy:0.5, radius: 0.7,
        fx:0.5, fy:0.5,
        stop:0 rgba(70, 70, 150, 255),  /* Màu sáng ở trung tâm */
        stop:1 rgba(10, 10, 50, 255)    /* Màu tối ở viền */
    );</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QWidget" name="gridWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>235</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:none;
border:none;</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="btn_edit">
       <property name="styleSheet">
        <string notr="true">    background-color: #3949ab;
    color: white;
    padding: 6px 12px;
    border: none;
    border-radius: 6px;
    font-weight: bold;
    min-width: 80px; /* ép chiều ngang tối thiểu */</string>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_delete">
       <property name="styleSheet">
        <string notr="true">    background-color: #3949ab;
    color: white;
    padding: 6px 12px;
    border: none;
    border-radius: 6px;
    font-weight: bold;
    min-width: 80px; /* ép chiều ngang tối thiểu */</string>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTextEdit" name="text_note">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>57</y>
     <width>381</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_date">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
